ci/min_deps.yaml

```diff
@@ -0,0 +1,19 @@
+name: min_deps
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - alembic
+  - astropy
+  - cartopy
+  - coverage
+  - numpy
+  - psycopg2
+  - pytest
+  - pyuvdata
+  - pyyaml
+  - pytest
+  - pytest-cov
+  - redis-py
```

Comment:
I'm a little confused as to why the minimum dependencies requires redis. You're not setting up a redis database in the test suite.

Reply:
Yeah, it's a little weird and I can change that if we want to. It's imported in several files and it was already listed as required in the `install_requires` section of `setup.py`, so I left it alone. But I did want to test that you can run without a redis server running.

Reply:
I see now. I've looked a little more thoroughly through the rest of the code base. I would be a lot more work adding in these checks for redis imports and erroring when calling redis required functions. Like you said we already have it set as a required dependency. I'm fine with booting this to an issue at least for now.